Changelog.md

```diff
@@ -10,8 +10,14 @@
 
 ## Unreleased
 
+- MSRV bumped to 1.52.
```

Comment:
This should go under a `## Misc Changes` section

Reply:
Yeah, that section is better than in the beginning. Sorry for noise, didn't think about it before